.github/pull_request_template.md

```diff
@@ -19,7 +21,9 @@ the terms of the Apache 2.0 license.
 `[Reviewer TODO: Verify that these criteria are met. Request changes if not]`
 
 - [ ] All commits in this PR are signed (`git commit -s`).
-- [ ] The reason for this PR is clearly provided (issue no. or explanation).
+- [ ] The reason for this PR is clearly provided (issue no.).
```

Comment:
I think that we're adding too many checkboxes here. The last two checkboxes can be merged into one.

Reply:
I will remove the `The reason for this PR is clearly provided (issue no.).` checkbox entirely as it is covered by the next one.